.gitmodules

```diff
@@ -25,3 +25,7 @@
 [submodule "sorc/gsi_monitor.fd"]
   path = sorc/gsi_monitor.fd
   url = https://github.com/NOAA-EMC/GSI-Monitor.git
+[submodule "sorc/nexus.fd"]
+  path = sorc/nexus.fd
+  url = https://github.com/NOAA-OAR-ARL/NEXUS.git
+  branch = feature/gcafs
```

Comment:
Do we need to get this branch into develop before the workflow PR is merged?

Reply:
Yes but that should be very simple to do